### PR TITLE
e2e: tests: kickoff test labels

### PR DESCRIPTION
**Pull request by fromanirh**
match e2e tests with test cases.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by Tal-or** [comment]
/retest
looks like a network glitch

**Comment by fromanirh** [comment]
/hold
let's review if this actually addresses the ful test case

File: test/e2e/install/install.go [hunk 1]
```diff
@@ -58,9 +58,8 @@ var _ = Describe("[Install] continuousIntegration", func() {
 	})
 
 	Context("with a running cluster with all the components", func() {
-		It("should perform overall deployment and verify the condition is reported as available", func() {
+		It("[test_id:47574] should perform overall deployment and verify the condition is reported as available", func() {
```
Comment (Tal-or) [inline test/e2e/install/install.go:61]: Do you mind explaining in the PR for the novices, what is the meaning of this number?

Comment (fromanirh) [reply test/e2e/install/install.go:61]: the test_id is the test case identifier as provided by our QE folks. We need them to match the e2e test with their test cases, so we know that if e2e test $WHATEVER passes (or not), then the testcase 12345 is satisfied (or not).

**Comment by fromanirh** [comment]
superceded by https://github.com/openshift-kni/numaresources-operator/pull/100

**Pull request closed without merging**